src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.common.hardware.MotorController;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.cameraserver.CameraServer;
+import edu.wpi.first.cscore.CvSink;
+import edu.wpi.first.cscore.CvSource;
+
+
+/** Add your docs here. */
+public class ClimbSubsystem extends SubsystemBase {
+  // Put methods for controlling this subsystem
+  // here. Call these from Commands.
+  
+  private MotorController m_climbMotorControllerOne;
+  private MotorController m_climbMotorControllerTwo;
+  private DigitalInput m_limitSwitch;
+  
+
+  // TODO: maybe add servos
+
+  public ClimbSubsystem() {
+    //One is left, two is right
+    m_climbMotorControllerOne = new MotorController("Climb Motor One", Constants.kClimbMotorOneIndex);
+
+    m_climbMotorControllerTwo = new MotorController("Climb Motor Two", Constants.kClimbMotorTwoIndex);
+    m_climbMotorControllerTwo.setInverted(true);
```

Comment:
Why do we set inverted here if the motors are taking the same action in the same direction?

Reply:
The motors for the Climer are going to face different directions meaning that one of the motors needs to be inverted